lib/pdfDownload.dart

```diff
@@ -34,7 +36,8 @@ class PdfDownload {
         if(resource['resourceType'] == "In Person")
           {
             // get the full address using addressString()
-            final ResourceDetail resourceDetail = ResourceDetail(resource: resource);
+            final ResourceDetail resourceDetail = ResourceDetail(
+                                      analytics: analytics, resource: resource);
             final String? fullAddress = resourceDetail.addressString();
```

Comment:
It seemed strange that we are threading an Analytics instance in here, and now I see there's a deeper issue -- the only reason we're using the ResourceDetail widget is to format the address as a string. Widgets really should only be used as Widgets. I think we can extract address formatting as a standalone function or something. Then we can remove Analytics.